src/Process/Builder/InitialProcessBuilder.php

```diff
@@ -36,30 +36,22 @@
 namespace Infection\Process\Builder;
 
 use Infection\Console\Util\PhpProcess;
-use Infection\Mutant\MutantInterface;
-use Infection\Process\MutantProcess;
 use Infection\TestFramework\AbstractTestFrameworkAdapter;
 use Symfony\Component\Process\Process;
 
 /**
  * @internal
  */
-class ProcessBuilder
+class InitialProcessBuilder
```

Comment:
To be consistent with the `runInitialTestSuite()` method, what do you think about renaming it to `InitialTestsProcessBuilder`, or `InitialTestRunProcessBuilder` to be consisten with this class'  method name?

`InitialProcess` is too common IMO